getting-started/client-blazor.md

```diff
@@ -1,21 +1,21 @@
 ---
-title: Client-side Blazor - Tutorial
-page_title: First Steps with Client-side Blazor
+title: Client-Side Blazor - Tutorial
+page_title: First Steps with Client-Side Blazor
 description: First Steps with UI for Blazor Client-side.
 slug: getting-started/client-side
 tags: get,started,first,steps,client
 published: true
 position: 1
 ---
 
-# First Steps with Client-side UI for Blazor
+# First Steps with Client-Side UI for Blazor
 
 This article explains how to get the Telerik UI for Blazor components in your **Client-side (WebAssembly)** Blazor project and start using them quickly. The process consists of the following steps:
 
 1. [Set Up a Blazor Project](#step-1---set-up-a-blazor-project)
```

Comment:
I find the TOC somewhat confusing: Step 0 is missing; Step 1 lists alternative methods and not required steps; Steps 2 and 3 list required steps. 
Maybe we can improve the list by removing the sub-steps and adding Step 0?

Reply:
We can't have the list start from zero (that "download" step was required by management rather recently), and renaming the headings will break all existing links that point to specific sections (we've linked them from tickets and forums). I added the sub-steps to illustrate options and actual steps in a rather lengthy process. I am fairly certain that the ToC was simpler in earlier versions and stretching it out with those details helped avoid confusion and improved reader satisfaction.

That said, if you think it will get better - feel free to restructure.